src/odbc/src/statement.cpp

```diff
@@ -551,6 +552,18 @@ SqlResult::Type Statement::InternalGetAttribute(int attr, void* buf, SQLINTEGER,
       break;
     }
 
+    case SQL_ROWSET_SIZE: {
+      SqlUlen* val = reinterpret_cast<SqlUlen*>(buf);
```

Comment:
I think SQL_ATTR_ROW_ARRAY_SIZE is also supposed to be a SQLULEN instead of SQLINTEGER...

Reply:
Changed. I have also changed the data types of `fetched` and `errors` and removed the negative value check for `fetched` in `InternalFetchRow`.